x-pack/platform/plugins/shared/inference/server/tracing/create_inference_active_span.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import apm from 'elastic-apm-node';
+import { isTracingSuppressed } from '@opentelemetry/core';
```

Comment:
For my own education: Will this method return `true` when in the future we've got OTel tracing, but he customer doesn't want the inference integration? Or vice-versa?

Reply:
it will return the value for the given context. so you can basically disable tracing for a specific endpoint, or the inference calls, but everything in that context will get dropped. so yes it could be used to disable inference-specific spans, but some events will be lost. It's mostly an optimisation to not do any cpu-intensive stuff when tracing is suppressed for that context.

Reply:
it's mostly used in instrumentations, eg: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/b3a70d7252438df0007bba7c4488c6dd72ebc42b/plugins/node/instrumentation-fs/src/instrumentation.ts#L18